src/engine/HedronEngine.ts

```diff
@@ -94,7 +95,14 @@ export class HedronEngine {
           paramValues[paramKey] = value
         })
 
-        sketchInstances[sketch.id].update({ deltaFrame: 1, params: paramValues })
+        const instance = sketchInstances[sketch.id]
+
+        if (instance.getPasses) {
+          instance.getPasses(debugScene).forEach((pass) => {
```

Comment:
Won't want to always get the passes every frame, but this was the easiest way in the moment (no need to worry about adding/removing, let sketches change their stack)

I'm thinking we refactor once we have sketch events?

Reply:
Works for now! 👍

Reply:
Btw you could also do:

```ts
instance.getPasses?.(debugScene).forEach((pass) => {
```

if you didn't want to use an if statement 😉